src/lib/auth.ts

```diff
@@ -26,36 +28,62 @@ export const config = {
           typeof email !== "string" ||
           email.length < 3 ||
           typeof password !== "string" ||
-          email.length < 3
+          password.length < 3
```

Comment:
We should enforce using password length with at least 8 characters.
```suggestion
          password.length < 8
```